content/en/docs/concepts/storage/projected-volumes.md

```diff
@@ -71,18 +71,29 @@ A container using a projected volume source as a [`subPath`](/docs/concepts/stor
 volume mount will not receive updates for those volume sources.
 {{< /note >}}
 
-## SecurityContext interactions
+### serviceAccountToken and securityContext
 
 The [proposal](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2451-service-account-token-volumes#proposal) for file permission handling in projected service account volume enhancement introduced the projected files having the the correct owner permissions set.
 
-### Linux
+#### Linux
 
-In Linux pods that have a projected volume and `RunAsUser` set in the Pod
-[`SecurityContext`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context),
-the projected files have the correct ownership set including container user
-ownership.
```

Comment:
I can't tell if this is an independent statement that still applies to other types of projected volumes like configmaps/secrets/downwardAPI

Reply:
I'm sure I knew at some point, but I've lost all context by now. I've reverted my change here and hope we can proceed with this PR as a simple addition of  detail.